File: app/src/main/java/com/concough/android/models/UserLogModel.java
package com.concough.android.models;

import java.util.Date;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

/**
 * Created by abolfazl on 7/16/17.
 */

public class UserLogModel extends RealmObject {
    @PrimaryKey
    public String uniqueId = "";

    public String username = "";
    public Date created = new Date();
    public String logType = "";
    public String extraData = "";
    public Boolean isSynced = false;
}

File: app/src/main/java/com/concough/android/models/UserLogModel.kt
package com.concough.android.models

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import java.util.*

/**
 * Created by abolfazl on 7/12/17.
 */
class UserLogModel: RealmObject() {
    @PrimaryKey
    var uniqueId: String = ""

    var username: String = ""
    var created: Date = Date()
    var logType: String = ""
    var extraData: String = ""
    var isSynced: Boolean = false
}